src/components/Button.js

```diff
@@ -1,17 +1,30 @@
-import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ name }) => (
-  <div>
-    <h1>{name}</h1>
-  </div>
-);
+export default function Button({
+  name, operator, actionBtn, onClick,
+}) {
+  return (
+    <button
+      className={`btn ${operator ? 'operatorBtn' : ''} ${
+        actionBtn ? 'actionBtn' : ''
+      }`}
+      value={name}
+      onClick={onClick}
+      type="button"
+    >
+      {name}
+    </button>
+  );
+}
+
 Button.defaultProps = {
```

Comment:
- [ ] This component should attach the `handleClick` to `onClick` event on Button.
- [ ] The `+/-` button is currently not functional. Please let's fix it.

Reply:
the +/_ button is working on my end

Reply:
The `+/-` button doesn't work currently in the live version of the app. Please make sure you commit changes made to the live app as well so that everything well synced.